docs/content/doc/help/seek-help.en-us.md

```diff
@@ -0,0 +1,28 @@
+---
+date: "2017-01-20T15:00:00+08:00"
+title: "Support Options"
+slug: "seek-help"
+weight: 10
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "help"
+    name: "Support Options"
+    weight: 20
+    identifier: "seek-help"
+---
+
+# Support Options
+
+- [#gitea on Freenode](http://webchat.freenode.net?nick=giteachat....&channels=%23gitea&prompt=1)
+- [Matrix](https://matrix.to/#/#gitea-dev:matrix.org)
+- [Discord](https://discord.gg/NsatcWJ)
```

Comment:
Please move discord up as that is our primary communication channel

Reply:
It's an "unordered" list. Being concerned about what's up at top feels a lot like bikeshedding to me.
Clearly this will be a whining point, though... so I'll change it regardless.